hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/tracing/Span.java

```diff
@@ -21,32 +21,36 @@
 
 import java.io.Closeable;
 
+/***
+ * This class is a wrapper class on top of opentelemetry Span class
+ * avoiding direct dependency on opentelemetry API
```

Comment:
afraid you will need a "." at the end to keep all versions of javadoc happy. longstanding PITA